vitrina/projects/forms.py

```diff
@@ -26,7 +32,29 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = Layout(
             Field('title', placeholder=_("Pavadinimas")),
             Field('description', placeholder=_("Aprašymas")),
+            Field('dataset'),
             Field('url', placeholder=_("Nuoroda į panaudojimo atvejį")),
             Field('image', placeholder=_("Pavadinimas")),
             Submit('submit', button, css_class='button is-primary')
         )
+
+
+class AddDatasetForm(ModelForm):
```

Comment:
We don't need this form and possibility to add datasets from project view, because there are too many datasets and this way of adding datasets to projects is not practical, nor convenient.

Instead, projects and datasets should be associated from datasets details view, see https://github.com/atviriduomenys/katalogas/issues/239.